.changeset/hungry-pandas-suffer.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": patch
+---
+
+Make send signatures configurable when Transmit in Contract Transmitter #changed
```

Comment:
Let's mark this #internal since it doesn't concern node ops
```suggestion
Make send signatures configurable when Transmit in Contract Transmitter #internal
```